### ISSUE TITLE
Fehler im Blog Eintrag und Suche

### ISSUE THREAD

**Issue by Nebukadneczar** [Werkstattl/OpenBlogware#12]
![fehler](https://github.com/user-attachments/assets/539ef0f8-fcc5-4a72-80fe-8bfa969d368e)

Es können keine Blöcke ausgewählt werden.

**Comment by 7underlines** [comment]
I can confirm that it is no longer working with Shopware 6.6.8.2. However, it does work with Shopware 6.6.6.0. I will try to pinpoint the cause of this issue.

**Comment by phit** [comment]
I pushed a fix to our repo, due to the full rename of this project I'm however unable to provide a meaningful PR for this repo at this time, we would have to migrate all our customers first

see https://github.com/Werkstattl/OpenBlogware/commit/b22d7cf77ed70cf68fdb9fe35b52ddce00ca19c9#diff-39adafce826a36cbb619aa1f7dce6d73548965868b32516af94192cff85036a0R160 specifically the page type change

**Comment by Nebukadneczar** [comment]
Vielen Dank

**Comment by 7underlines** [comment]
@Nebukadneczar Could you please try it again with the new v4.1.0 version? This issue should be resolved. However, you’ll need to fill in the basic "Blog Detail" information first. 

Thank you very much, @phit!

**Comment by Nebukadneczar** [comment]
> @NebukadneczarKönnten Sie es bitte noch einmal mit der neuen Version v4.1.0 versuchen? Dieses Problem sollte behoben sein. Sie müssen jedoch zuerst die grundlegenden „Blog-Details“-Informationen eingeben.
> 
> Vielen Dank,@phit!

Ja, ich teste das jetzt über die Feiertage und werde dann reportieren.

**Comment by Nebukadneczar** [comment]
I have the following setup in the test environment:

Shopware 6.6.9.0 and tried out different themes
PHP 8.3.14
OpenBlogware 4.1.0

Installation goes smoothly! Bug has been fixed. 

If the search box is activated in the configuration, results from articles and blogs are displayed, but these cannot be clicked on (https://xxxx.com/Blog/?p=3#v-pills-product) **Product result (https:// xxxxx.com/#v-pills-blog)** blog result and therefore cannot be accessed. The following phenomenon is particularly noteworthy: If there is a hit in the search with identical terms, the block entries are not displayed. So: I'm looking for the term "Stable" which appears in the title of a blog post, as well as a product that is called "Stable". So only the article will be displayed!

**Comment by 7underlines** [comment]
I resolved the issue with mixed search results in v4.1.3. Please give it a try.

The #v-pills links should primarily display the number of items found. You can see an example in the screenshot below, showing 1 result for products and 1 result for blog search.
 
![Bildschirmfoto vom 2025-01-01 20-07-59](https://github.com/user-attachments/assets/a9a30370-2470-4167-ae50-e4385696f8cd)

**Comment by Nebukadneczar** [comment]
Das Problem wurde behoben , ich habe es

Shopware 6.6.9.0 und verschiedene Themes ausprobiert
PHP 8.3.14
OpenBlogware 4.1.3

Ich habe die alte Version entfernt, ohne die Daten zu löschen, und dann die 4.1.3. installiert. Die Installation verläuft reibungslos! Fehler wurde behoben. Danke
